just-install.json

```diff
@@ -391,6 +391,14 @@
       },
       "version": "5.0.4"
     },
+    "media-player-classic-home-cinema": {
```

Comment:
I was wondering that maybe we can use `mpc-hc` here since it looks like the software is mostly known by this name.